### DISCUSSION TITLE
Questions for gas fee & using ERC-20 token for polygon-edge chain

### DISCUSSION THREAD

**Discussion by KJHBC** [0xPolygon/polygon-edge#560] [Q&A]
Following official documentation for local chain setup, I have 2 questions.

### Q1. 

How can I set transaction gas fee for local polygon-edge chain? 
(cannot find settings for gas fee at documentation)
Should I include gas fee settings when creating genesis file? 

### Q2. (related to Q1)

Could I force to pay custom ERC-20 token deployed on local chain for gas fee, 
instead of native token - also for staking & unstaking?

**Comment by laviniat1996** [comment]
Hello @KJHBC ,

Thank you for your questions :).

**To answer them:**

**Q1:** You cannot set the gas fee for a certain transaction. What you can do is to set the minimum gas limit, by using the `--price-limit` flag. This setting is not in the genesis file, it is on the server, so it can be changed each time you start a new node.
See more about it here (scroll down towards the end of the page): https://edge-docs.polygon.technology/docs/get-started/set-up-ibft-locally

**Q2:**:  The token for the gas fee has to be the native token of that network. You cannot deploy a custom token for the gas fees.
